FluentPDO/BaseQuery.php

```diff
@@ -114,9 +117,15 @@ public function execute() {
 		}
 
 		$time = microtime(true);
-		if ($result && $result->execute($parameters)) {
+		if ($result && @$result->execute($parameters)) {
 			$this->time = microtime(true) - $time;
-		} else {
+		} else { // Oops, something went wrong with the query.
+            $info = $result->errorInfo();
+            $this->last_error = new FluentQueryException(
```

Comment:
@Etenil Either throw the exception or refactor FluentQueryException to FluentQueryError or similar